github/src/main/kotlin/gropius/sync/github/repository/RepositoryInfoRepository.kt

```diff
@@ -0,0 +1,20 @@
+package gropius.sync.github.repository
+
+import gropius.sync.github.model.RepositoryInfo
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository
+import org.springframework.stereotype.Repository
+
+/**
+ * Mapping of a single repository from neo4j to github
```

Comment:
Maybe something along the lines of "repository for retrieving the mappings..."

This comment implies this is the model class.

Reply:
93208eeec463b1a5f6cd2de7a2af126618451613